tasks/check_execute.rb

```diff
@@ -0,0 +1,32 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
Add docs

Reply:
What docs would be useful that aren't handled by the JSON task definition? The Ruby code is pretty basic.